### ISSUE TITLE
azure-kusto-data 4.5 helpers fail constructing dataframe of .show materialized-views

### ISSUE THREAD

**Issue by borlaugen** [Azure/azure-kusto-python#542]
```python
materialized_views_response = self.kusto_client.execute(
    self._database, ".show materialized-views"
)
materialized_tables = dataframe_from_result_table(
    materialized_views_response.primary_results[0]
)

```
The above code will with version 4.5 raise a  KeyError exception on key 'time' since the LookBack column has ColumnType 'time'.

This is most likely introduced by commit 4f9af0f5926d73412e8865d3edd5a7e469efee01

![image](https://github.com/Azure/azure-kusto-python/assets/59911423/2e69b203-32b3-47e1-8424-3af6cdb6faf3)

**Comment by AsafMah** [comment]
Thanks, we will fix this.

As a workaround for now you can pass a converter like so:
```
            from azure.kusto.data.helpers import parse_timedelta
            data_frame = dataframe_from_result_table(data, converters_by_type={"time": lambda col, df: df[col].apply(parse_timedelta)})
```

**Comment by AsafMah** [comment]
Fixed in 4.5.1
